scanner/src/main/kotlin/scanners/ScanCode.kt

```diff
@@ -72,35 +86,39 @@ object ScanCode : LocalScanner() {
         return scancodeDir
     }
 
+    override fun getConfiguration() = DEFAULT_OPTIONS.toMutableList().run {
+        add(OUTPUT_FORMAT_OPTION)
+        joinToString(" ")
+    }
+
     override fun getVersion(executable: String) =
             getCommandVersion(scannerPath.absolutePath, transform = {
                 // "scancode --version" returns a string like "ScanCode version 2.0.1.post1.fb67a181", so simply remove
                 // the prefix.
                 it.substringAfter("ScanCode version ")
             })
 
-    override fun scanPath(path: File, resultsFile: File): Result {
+    override fun scanPath(path: File, resultsFile: File, provenance: Provenance, specification: ScannerSpecification)
+            : ScanResult {
         val options = DEFAULT_OPTIONS.toMutableList()
+
         if (log.isEnabledFor(Level.DEBUG)) {
```

Comment:
Shouldn't this be part of the scanner configuration, too?

Reply:
My assumption was that the ScanCode results file is the same with and without debug, so this option is not relevant for the result. Is that correct?

Reply:
I believe so, yes. We should then clearly document that we only store those configurations settings as part of the results that have an impact on the results, and e.g. settings to specify the output file name are omitted.

Reply:
I have improved this now, please check again.